lib/main-menu.js

```diff
@@ -136,6 +143,33 @@ var view = {
       click: function () {
         BrowserWindow.getFocusedWindow().toggleDevTools()
       }
+    },
+    {
+      type: 'separator'
+    },
+    {
+      label: 'Next Note',
+      accelerator: 'Control + J',
+      click: function () {
+        mainWindow.webContents.send('list:next')
+      }
+    },
+    {
+      label: 'Prior Note',
```

Comment:
I should fix it to prev because the opposite word of `next` seems `previous`. 
http://www.wordhippo.com/what-is/the-opposite-of/next.html